### PR TITLE
Add support for "collapse to definition"

### PR DESCRIPTION
**Pull request by vasily-kirichenko**

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs [hunk 1]
```diff
@@ -18,53 +18,102 @@ open Microsoft.VisualStudio.FSharp.Editor.Structure
...
+        | Scope.YieldOrReturnBang
+        | Scope.TryWith -> BlockTypes.Expression
+        | Scope.Do -> BlockTypes.Statement
+        | Scope.While
+        | Scope.For -> BlockTypes.Loop
+        | Scope.HashDirective -> BlockTypes.PreprocessorRegion
+        | Scope.Comment
+        | Scope.XmlDocComment -> BlockTypes.Comment
+
+    let isAutoCollapsible = function
```
Comment (KevinRansom) [inline vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs]: @vasily-kirichenko 
Just a code formatting thing ... the first two times I looked at this I thought that the only return value was false.  Perhaps if we put a blank line between the true and false blocks I would have caught the trues earlier.  But perhaps that is just me.

Kevin

Comment (KevinRansom) [inline vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs]: thanks

Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs]: You are right. Done.

**Comment by KevinRansom** [comment]
@vasily-kirichenko 
Thank you for this

**Pull request merged**